lambda/utils.js

```diff
@@ -142,7 +172,11 @@ module.exports.getLambdaPower = async(lambdaARN) => {
     };
     const lambda = utils.lambdaClientFromARN(lambdaARN);
     const config = await lambda.send(new GetFunctionConfigurationCommand(params));
-    return config.MemorySize;
+    return {
+        power: config.MemorySize,
+        // we need to fetch env vars only to add a new one and force a cold start
+        description: config.Description,
```

Comment:
As mentioned in another comment, this implementation of `getLambdaPower` has become very similar to `getLambdaConfig` below. It used to make more sense because it was a specialized version that only cared about `config.MemorySize` and returned a single value. But now it returns a structure with various config parameters, so I'd consider merging `getLambdaPower` and `getLambdaConfig` back into one single implementation where we fetch all the config parameters we need.